checker/src/abstract_value.rs

```diff
@@ -1626,9 +1626,6 @@ impl AbstractValueTrait for Rc<AbstractValue> {
             Expression::Reference(path) | Expression::Variable { path, .. } => {
                 path.is_rooted_by_zeroed_heap_block()
             }
-            Expression::UnknownTagCheck { operand, .. } => {
```

Comment:
I'm left wondering why this is no longer needed.

Reply:
My understanding of `AbstractValue::is_contained_in_zeroed_heap_block` is that it checks if the value itself is contained in a zeroed heap block. It is not the case for unknown tag checks.